src/getters/crunchbase.py

```diff
@@ -0,0 +1,132 @@
+"""
+discovery_utils.src.getters.crunchbase.py
+
+Module for easy access to downloaded CB data on S3.
+
+"""
+import pandas as pd
+import boto3
+import os
+from dotenv import load_dotenv
+from botocore.client import BaseClient
+from src.utils import s3 as s3
+from src.utils import timestamps as ts
+from datetime import datetime, timedelta
+import logging
+import json
+
+# logging level
+logging.basicConfig(level=logging.INFO)
+
+load_dotenv()
+
+# Retrieve AWS file information from environment variables
+BUCKET_NAME_RAW = os.getenv("BUCKET_NAME_RAW")
+FILE_NAMES_RAW = json.loads(os.getenv("FILE_NAMES_RAW", '[]'))
+S3_PATH_RAW = os.getenv("S3_PATH_RAW")
+# Retrieve AWS credentials from environment variables
+AWS_ACCESS_KEY = os.getenv("AWS_ACCESS_KEY")
+AWS_SECRET_KEY = os.getenv("AWS_SECRET_KEY")
+
+
+# Left these here for when they are used using AWS CLI
+# os.environ["AWS_ACCESS_KEY"]
+# os.environ["AWS_SECRET_KEY"]
+
+
+# VERSATILE FUNCTION FOR GENERALISED USE
+
+def _s3_client(aws_access_key_id: str = AWS_ACCESS_KEY,
+               aws_secret_access_key: str = AWS_SECRET_KEY
+               ) -> BaseClient:
+    """Initialize S3 client"""
+    S3 = boto3.client(
+        "s3", aws_access_key_id, aws_secret_access_key
+        )
+    return S3
+
+def get_table(timestamp: str,
```

Comment:
This is the main getter function. It would be great to know if this can be improved.